entity/contents/solid_stroke_contents.cc

```diff
@@ -27,6 +29,30 @@ const Color& SolidStrokeContents::GetColor() const {
   return color_;
 }
 
+std::optional<Rect> SolidStrokeContents::GetCoverage(
+    const Entity& entity) const {
+  auto path_coverage = entity.GetPathCoverage();
+  if (!path_coverage.has_value()) {
+    return std::nullopt;
+  }
+
+  Scalar max_radius = 0.5;
+  if (cap_ == Cap::kSquare) {
+    max_radius = max_radius * kSqrt2;
+  }
+  if (join_ == Join::kMiter) {
+    max_radius = std::max(max_radius, miter_limit_ * 0.5f);
+  }
+  Vector2 max_radius_xy = entity.GetTransformation().TransformDirection(
+      Vector2(max_radius, max_radius) * stroke_size_);
+
+  return Rect(
+      Point(std::floor(path_coverage->origin.x - max_radius_xy.x),
```

Comment:
Why are you doing a floor/ceil here? Shouldn't it be up to the caller whether they want the bounds to be rounded out?

Reply:
Good point, thanks -- removed this.